committeeoversightapp/views.py

```diff
@@ -74,10 +75,19 @@ class EventList(LoginRequiredMixin, TemplateView):
 
 class EventListJson(BaseDatatableView):
     """ Uses django-datatables-view for server-side DataTable processing."""
-    model = Event
-
-    # define the columns that will be returned
-    columns = ['updated_at', 'name', 'start_date', 'id', 'id']
+    model = HearingEvent
+
+    # Define the columns that will be returned. These need to match attributes
+    # of the model but most will be calculated later, so 'id' here is a
+    # placeholder.
+    columns = [
+        'start_date',
+        'name',
+        'id', # committees
```

Comment:
Are you required to include a value here, or can it be an empty string?

Reply:
This does work with an empty string. Good idea!